.github/workflows/ci.yml

```diff
@@ -19,8 +19,20 @@ jobs:
       # Required to create loop devices.
       options: --privileged
     steps:
-    - uses: actions/checkout@v2
-    - name: Run storage tests
+    - uses: actions/checkout@v3
+    - name: Run storage tests not marked as root as vdsm user
+      run: ./automation/tests-storage.sh vdsm -u=vdsm
+  test-storage-root:
+    env:
+      TRAVIS_CI: 1
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/ovirt/vdsm-test-centos-8
```

Comment:
I'd really love to limit the amount of images we manage. For CI leveraging the https://github.com/oVirt/buildcontainer would be best, both for building and running tests

Reply:
Sounds reasonable. We can bring this up in the next vdsm meeting, but changing the image is out of the scope of this patch. For this change, the image of the new job is consistent with the other jobs in the CI.